src/interactions/map_arcade/jessie.interaction.js

```diff
@@ -0,0 +1,24 @@
+import { displayDialogue } from '../../utils';
+
+import { npcInteractionHandler } from '../handler.interactions';
+
+import { conversationJessie, jessie } from '../../constants';
+
+export const interactionWithJessie = (player, k, map) => {
+    player.onCollide('jessie', () => {
+        player.isInDialog = true;
+
+        npcInteractionHandler(player, jessie.name, k, () => {
```

Comment:
```npcInteractionHandler``` is being used incorrectly in this exported function. Please look at bruno.interaction.js